RankensteinApp/App.js

```diff
@@ -0,0 +1,183 @@
+               <Text style = {styles.submitButtonText}> Login </Text>
+        </TouchableOpacity>
+		<TouchableOpacity
+               style = {styles.submitButton}
+               onPress = {
+                  () => this.props.navigation.navigate('Create',{title: 'WHATEVER'})
+               }>
+               <Text style = {styles.submitButtonText}> Create New Account </Text>
+        </TouchableOpacity>
+      </View>
+    );
+  }
+}
+
+class HomeScreen extends React.Component {
+	render() {
+    return (
+      <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
+        <Text>Details Screen</Text>
+      </View>
+    );
+  }
+}
+
+class CreateAccountScreen extends React.Component {
+	state = {
+      username: '',
+      password: '',
+	  verifypass: '',
+	  email: '',
+  }
+  handleUsername = (text) => {
+      this.setState({ username: text })
+  }
+  handlePassword = (text) => {
+      this.setState({ password: text })
+  }
+  handleVerifypass = (text) => {
+	  this.setState({verifypass: text})
+  }
+  handleEmail = (text) => {
+	  this.setState({email: text})	
+  }
+  createAccount = (username, pass, verifypass, email) => {
+      alert('email: ' + email + ' username: ' + username + ' password: ' + pass + ' verify password: ' + verifypass)
+  }
+  render() {
+    return (
+      <View style={styles.container}>
+        <TextInput style = {styles.input}
+           underlineColorAndroid = "transparent"
+           placeholder = "Email"
+           placeholderTextColor = "#fff"
+           autoCapitalize = "none"
+           onChangeText = {this.handleEmail}/>
+		<TextInput style = {styles.input}
+           underlineColorAndroid = "transparent"
+           placeholder = "Username"
+           placeholderTextColor = "#fff"
+           autoCapitalize = "none"
+           onChangeText = {this.handleUsername}/>
+		<TextInput style = {styles.input}
+		   secureTextEntry={true}
+           underlineColorAndroid = "transparent"
+           placeholder = "Password"
+           placeholderTextColor = "#fff"
+           autoCapitalize = "none"
+           onChangeText = {this.handlePassword}/>
+		<TextInput style = {styles.input}
+		   secureTextEntry={true}
+           underlineColorAndroid = "transparent"
+           placeholder = "Verify Password"
+           placeholderTextColor = "#fff"
+           autoCapitalize = "none"
+           onChangeText = {this.handleVerifypass}/>
+		<TouchableOpacity
+               style = {styles.submitButton}
+               onPress = {
+                  () => this.createAccount(this.state.username, this.state.password,this.state.verifypass,this.state.email)
+               }>
+               <Text style = {styles.submitButtonText}> Create Account </Text>
+        </TouchableOpacity>
+      </View>
+    );
+  }
+}
+
+const SN = createStackNavigator(
+  {
+  	Login: LoginScreen,
+  	Home: HomeScreen,
+  	Create: CreateAccountScreen,
+  },
+  {
+  	initialRouteName: 'Login',
```

Comment:
Actually a splash screen might be a good idea, though that can be a future change